che-theia-init-sources.yml

```diff
@@ -12,6 +12,7 @@ sources:
     - extensions/eclipse-che-theia-about
     - extensions/eclipse-che-theia-preferences-provider-extension
     - extensions/eclipse-che-theia-git-provisioner
+    - extensions/che-theia-logging
```

Comment:
maybe it makes sense to align with other extensions, so `eclipse-che-theia-logging` instead of `che-theia-logging`
wdyt?

Reply:
Updated